tiers.js

```diff
@@ -245,6 +251,26 @@ function load_tierlist(serialized_tierlist) {
 	unsaved_changes = false;
 }
 
+// Returns the supplied item's index within a row
+function get_item_index(elem) {
+	let rows = Array.from(tierlist_div.querySelectorAll(".row"));
+	let parent_div = elem.parentNode.parentNode.parentNode;
+	let idx = rows.indexOf(parent_div);
+	let item_index;
+
+	if (rows[idx] !== undefined) {
+		let image_node_list_count = rows[idx].querySelectorAll("img").length;
+		let image_node_list = rows[idx].querySelectorAll("img");
+		for (let i = 0; i < image_node_list_count; i++) {
+			if (image_node_list[i] == elem) {
+				item_index = i;
+				break;
```

Comment:
Here you can directly `return i` instead of breaking and returning outside the `for`